android-lib/src/main/java/com/kinvey/android/cache/RealmCacheManager.java

```diff
@@ -54,6 +56,18 @@
     private static final Object LOCK = new Object();
 
 
+    public RealmCacheManager(byte[] encryptionKey, Client client){
+        this.encryptionKey = encryptionKey;
+        this.client = client;
+        this.context = client.getContext();
+    }
+
+    public RealmCacheManager(byte[] encryptionKey, String prefix, Client client){
+        this.encryptionKey = encryptionKey;
+        this.client = client;
+        this.context = client.getContext();
+    }
+
```

Comment:
What is the use of `prefix` in this constructor?

Reply:
Prefix is used to create realmConfiguration: "realmConfiguration = new RealmConfiguration.Builder().name(prefix + "_" + getClientHash())". At the current moment the prefix isn't saved (wasn't saved previously). It should be investigated more.